packages/espressocash_app/lib/core/feature_flags.dart

```diff
@@ -18,4 +19,11 @@ class FeatureFlagsManager {
     );
     await _remoteConfig.fetchAndActivate();
   }
+
+  bool isExperimental({required String address}) => _remoteConfig
```

Comment:
The app shouldn't get the list of addresses directly. Instead, there's a parameter created in `RemoteConfig` console, and here you just get the bool value, so it will be just:

```dart
bool get isSomeFeatureEnabled => _remoteConfig.getBool('isSomeFeatureEnabled');
```